proofpoint_tap/setup.py

```diff
@@ -3,7 +3,7 @@
 
 
 setup(name="proofpoint_tap-rapid7-plugin",
-      version="2.0.0",
+      version="2.0.1",
```

Comment:
It's a major version bump. You're requiring inputs that weren't required before.